src/code/ucode_disas.c

```diff
@@ -104,11 +109,12 @@ const char* UCodeDisas_ParseCombineColor(u32 value, u32 idx) {
     return ret;
 }
 
-const char* UCodeDisas_ParseCombineAlpha(u32 value, u32 idx) {
+const char* UCodeDisas_CombineAlphaStr(u32 value, u32 arg) {
```

Comment:
Maybe use `Get` since the string is returned instead of directly printed like with `UCodeDisas_GeometryModeStr` and others?